rs/cli/src/commands/update_authorized_subnets.rs

```diff
@@ -0,0 +1,158 @@
+use std::{
+    collections::BTreeMap,
+    fs::File,
+    io::{BufRead, BufReader},
+    path::PathBuf,
+    sync::Arc,
+};
+
+use clap::{error::ErrorKind, Args};
+use ic_management_types::Subnet;
+use ic_registry_subnet_type::SubnetType;
+use ic_types::PrincipalId;
+use itertools::Itertools;
+use log::info;
+
+use crate::ic_admin::{ProposeCommand, ProposeOptions};
+
+use super::ExecutableCommand;
+
+const DEFAULT_CANISTER_LIMIT: u64 = 60_000;
+const DEFAULT_STATE_SIZE_BYTES_LIMIT: u64 = 322_122_547_200; // 300GB
+
+#[derive(Args, Debug)]
+pub struct UpdateAuthorizedSubnets {
+    /// Path to csv file containing the blacklist.
+    #[clap(default_value = "./facts-db/non_public_subnets.csv")]
```

Comment:
Is this going to be relative path to the current dir? Can we do something better, e.g. embed default *contents* of the file, and allow providing another file path on the command line, in which case we don't need the default value.

Reply:
Its a bit weird because if you don't change the file, running it doesn't make sense, and if you have a random file on your drive it could have a mistake. My goal was to try and say "we expect you to be in the dre repo and change a file, meaning you will have a diff and remember to commit and update the file on repo". All of the solutions have some flaws, but we can for sure embed it in build time.